Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -114,6 +117,9 @@ namespace Multiplayer
     AZ_CVAR(AZ::TimeMs, bg_captureTransportPeriod, AZ::TimeMs{1000}, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
         "How often in milliseconds to record transport metrics.");
 
+    AZ_CVAR(bool, bg_parallelNotifyPreRender, true, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "bg_useJobsNotifyPreRender");
```

Comment:
nit: could we add a meaningful description to this cvar? How will others know how to use this setting?

Reply:
We should default it to false as well for safety and turn it on in mps depot for testing.